requirements/prod.in

```diff
@@ -1,4 +1,6 @@
-# These packages are installed from PyPI.
+django==3.2.20
```

Comment:
```suggestion
django==3.2.21
```

Updated earlier today in 0c9205d5bb68655528c9d63eb75eb2eec3ddbeff.

Reply:
Yeah, will get that when resolving conflicts when rebasing on main.

Reply:
Superseded by further Django updates.